### DISCUSSION TITLE
maim piped to xclip works from command line, but not from bindsym or any file

### DISCUSSION THREAD

**Discussion by Runkli** [i3/i3#4967] [Q&A]
I'm able to pipe the output of maim to xclip when I run the command from command line (although it fails to detect a compositor and disables OpenGL), but this does not work as a bindsym from config. This also doesn't work when I put the command into a file and execute it from command line.

Here is the command:
```
bindsym $mod+Shift+s exec --no-startup-id maim --select | xclip -selection clipboard -t image/png
```

What may be the problem here?

**Comment by Airblader** [comment]
Use

```
bindsym --release $mod+Shift+s
```
